go.mod

```diff
@@ -33,6 +33,88 @@ require (
 	sigs.k8s.io/kind v0.11.1 // Do not remove, not used by code but used by build
 )
 
+require (
+	cloud.google.com/go v0.81.0 // indirect
```

Comment:
Woah, did something change with go 1.17 that adds all these `indirect` entries now?

Reply:
it's shifted things around

Reply:
and added a whole bunch more...

the second "require" section is the set of indirect imports

Reply:
https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17